### PR TITLE
Add note about pages-created count

### PR DESCRIPTION
**Pull request by samwilson**
Also fixes up some other documentation issues.

Bug: https://phabricator.wikimedia.org/T169955

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -198,6 +198,7 @@
   "pager-older-n": "{{PLURAL:$1|older 1|older $1}}",
   "pages": "Pages",
   "pages-created": "Pages created",
+  "pages-created-tooltip": "Total of all pages created, inlcuding those created as a redirect when moving a page.",
```
Comment (ricordisamoa) [inline i18n/en.json]: typo?

Comment (MusikAnimal) [inline i18n/en.json]: qqq? :)

File: app/Resources/views/editCounter/general_stats.html.twig [hunk 2]
```diff
@@ -126,7 +126,12 @@
                 <td>{{ ec.averageRevisionsPerPage }}</td>
             </tr>
             <tr>
-                <td>{{ msg('pages-created') }}</td>
+                <td>
+                    <a class="glyphicon glyphicon-info-sign" target="_blank" title="{{ msg('pages-created-tooltip') }}"
+                       href="https://xtools.readthedocs.io/en/latest/tools/editcounter.html#general-statistics">
+                    </a>
+                    {{ msg('pages-created') }}
+                </td>
```
Comment (MusikAnimal) [inline app/Resources/views/editCounter/general_stats.html.twig]: This works but the only way to see the translated message is to hover and _wait_ over the info icon. Most will click on the link which is only in English. If you want we could use our own tooltip system which will make the translatable message more prominent https://github.com/x-tools/xtools/blob/master/app/Resources/views/macros/forms.html.twig#L5-L11

**Review by MusikAnimal — CHANGES_REQUESTED** [review]
needs qqq mainly, but also we might look into using a tooltip rather than using the `title=` attribute

File: i18n/en.json (hunk 1 above)
Comment (samwilson) [reply i18n/en.json]: oops! done.

File: app/Resources/views/editCounter/general_stats.html.twig (hunk 2 above)
Comment (samwilson) [reply app/Resources/views/editCounter/general_stats.html.twig]: I've switched to the `tooltipcss` system, and added a couple of bits to that system; see what you think.

**Comment by samwilson** [comment]
Please re-review.

**Comment by MusikAnimal** [comment]
Good stuff! I add some minor CSS changes

**Pull request merged**